### DISCUSSION TITLE
sice of the plug-in

### DISCUSSION THREAD

**Discussion by banalytic** [publicsamples/Oi-Grandad#20] [Ideas]
Today I found oi, grandad through a recommendation and I'm really excited. thank you for this great plugin. I was just wondering if you can scale the size of the plugin? on a larger monitor it is unfortunately difficult to see in the daw. maybe it would be possible to implement different levels of magnification?
Thank you very much and good luck :)

**Comment by RickTaylor79** [comment]
Sure, this is pretty easy to do, I'll look into the for the next release.

**Reply by banalytic** [reply]
thanks for your quick reply. that sounds really good!
Free scalability would be cool, if that weren't so easy, maybe in 25% steps up to 250%?

**Reply by RickTaylor79** [reply]
Okay, 1.1 includes a UI zoom option (press the presets & settings button) with a few different scalings from 50% all the way up to 250%.

Certain UI elements are raster images, so expect a bit of fuzziness at higher settings. 

Thanks for the suggestion!

**Reply by banalytic** [reply]
wow! that's amazing, thank you sooo much for the fast implementation!

Now - as I can see the plugin - I had time t play around. I have another question about the Amp-Envelope. Decay/release is combined?! Is it normal that there is just a short decay of about 1 sec or maybe less?!

**Reply by RickTaylor79** [reply]
Effectively the envelope has no release stage. It's all down to the slightly unusual way granular has been implemented in HISE (the framework I used to build this). One of the goals for oi grandad! v2 is a new granular engine that gets round this problem. For the time being, try making use of the delays and with a bit of creative patching, the modulators can be used as multi-stage envelopes.

**Reply by banalytic** [reply]
do you have paypal?… would like to pay a coffee 😌

**Reply by RickTaylor79** [reply]
Thanks, much appreciated! Take a look at my store, if something looks good I'll be sure to spend it on coffee!

https://modularsamples.gumroad.com

**Comment by banalytic** [comment]
oh. I understand. nevertheless this plugin is wonderful. really like it! and thanks for the hints!
did you ever think about programming in max/msp or now: rnbo. https://cycling74.com/

**Reply by RickTaylor79** [reply]
I've never used MAX, but was taught some C-sound many years ago. HISE just integrated rnbo and FAUST recently, so I'm gonna try and get my head round those at some point.
